### ISSUE TITLE
pypy on freebsd

### ISSUE THREAD

**Issue by Azulinho** [pypy/pypy#5159]
any plans to build a pypy binary for freebsd ?

**Comment by mattip** [comment]
No, we don't have the bandwidth to do this. Isn't there a forum to request packages for freebsd?

**Comment by mattip** [comment]
No response, so closing. Please reopen with more information if there is anything we can do to help the freebsd package managers get a PyPy package built.
